src/controllers/BookingController.js

```diff
@@ -0,0 +1,27 @@
+import Response from '../helpers/Response';
+import AccommodationService from '../services/BookingService';
+
+/**
+ * @exports
+ * @class CommentController
+ */
```

Comment:
I think you could have used `BookingController` instead of `CommentController` in JSdoc